### ISSUE TITLE
Validate Response status codes to be 3 digit

### ISSUE THREAD

**Issue by evertonmc** [IBM/openapi-validator#163]
HTTP status codes are supposed to be [3 digit ](https://tools.ietf.org/html/rfc7231#section-6) and and it gets through the validator

i.e
```
        "responses": {
          "0": {
            "description": "Response on failure.",
            "schema": {
              "$ref": "#/definitions/ErrorReturnObject"
            }
          },
```
autorest validates to 
"Parsing error(s): Property '0' has not been defined and the schema does not allow additional properties. Path 'paths['/plans'].post.responses.0',

**Comment by dpopp07** [comment]
I agree, this should be caught. Thanks for the issue, we will look into it.

**Comment by barrett-schonefeld** [comment]
Closing because this is now covered by the `oas3-schema` rule
